geomstats/geometry/discrete_curves.py

```diff
@@ -1694,17 +1665,17 @@ def path(t):
             t = gs.cast(t, gs.float32)
             t = gs.to_ndarray(t, to_ndim=1)
             t = gs.to_ndarray(t, to_ndim=2, axis=1)
-            new_initial_curve = gs.to_ndarray(initial_curve, to_ndim=curve_ndim + 1)
+            new_initial_point = gs.to_ndarray(initial_point, to_ndim=point_ndim + 1)
             new_initial_tangent_vec = gs.to_ndarray(
-                initial_tangent_vec, to_ndim=curve_ndim + 1
+                initial_tangent_vec, to_ndim=point_ndim + 1
```

Comment:
Thank you so much for the renamings curve -> point, it was overdue!

We should also think about a strategy on how to handle these APIs:
- from the codebase perspective, we want to use `point` everywhere,
- from the user's perspective, they probably want to see `curve`
---> We could write an "API remapping" for the manifolds where this happens.

Reply:
Yes I was also thinking of a similar "API remapping" solution, I completely agree it would be helpful.